mcstatus/protocol/connection.py

```diff
@@ -409,25 +398,24 @@ async def read_buffer(self) -> Connection:
         return result
 
 
-class BaseConnection:
+class BaseConnection:  # todo: maybe use abstract class?
```

Comment:
```suggestion
class BaseConnection:
```

This isn't related to docstrings, if you want to see this, make a github issue instead. I don't like the idea of keeping tons of TODO comments directly in code, let's use the bug tracker on github directly, where we can discuss whether this makes sense, who will implement it, and how it could be implemented.

Reply:
Created #459, removed [here](https://github.com/py-mine/mcstatus/compare/b4c7e9763c5d6f7138586c81b52947c1894b9073..bb194c20f1882cd4832076a1360070ec9bffce64).